src/js/tech/tech.js

```diff
@@ -238,6 +238,14 @@ class Tech extends Component {
    * @method dispose
    */
   dispose() {
+    // clear out text tracks because we can't reuse them between techs
```

Comment:
Ok so help me understand what this means for users. What are caveats of using text tracks that we need to tell them?